src/stylesheets/elements/_table.scss

```diff
@@ -4,7 +4,7 @@ table {
   min-width: 100%;
 
   thead {
-    tr {
+    th {
```

Comment:
Because the white background will override the gray table header unless this is changed to `th`.